library/src/main/scala/scala/scalajs/js/Date.scala

```diff
@@ -48,141 +48,144 @@ class Date extends js.Object {
    *
    * MDN
    */
-  def getFullYear(): Int = js.native
+  def getFullYear(): Double = js.native
```

Comment:
I think the `js.Date` class deserves a general comment on why all the members here are typed as `Double`. IMO the MDN comment is *very* misleading: "that represents a single moment in time" is *not* the only thing it does (apparently). It can also represent an invalid time. We should document this.

Reply:
Done.